sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -62,6 +63,15 @@ protected override bool IsValidMemberForOverload(MemberDeclarationSyntax member)
             }
             return true;
         }
+
+        protected override bool IsStatic(MemberDeclarationSyntax member)
+        {
+            if (member is BaseMethodDeclarationSyntax declaration) //Method or Constructor
```

Comment:
[minor] we all agreed to follow Microsoft coding conventions unless we explicitly defined other conventions. 

> We’re all aligned to follow the Microsoft convention
([notes](https://docs.google.com/document/d/1W40QrhWslS-8dxx7Fu316qv6cLy1si9XQS3RgyVyDQE/edit)) 

Please read [Commenting Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#commenting-conventions) (which are aligned with existing SonarSource conventions) and put the comment on a separate line, with a space after the `//`